datahub-web-react/src/app/ingest/source/IngestedAssets.tsx

```diff
@@ -0,0 +1,174 @@
+import { ANTD_GRAY } from '../../entity/shared/constants';
+import { formatNumber } from '../../shared/formatNumber';
+import { Message } from '../../shared/Message';
+import { capitalizeFirstLetterOnly, pluralize } from '../../shared/textUtil';
+
+const ENTITIES_WITH_SUBTYPES = new Set([
+    EntityType.Dataset.toLowerCase(),
+    EntityType.Container.toLowerCase(),
+    EntityType.Notebook.toLowerCase(),
+]);
+
+type Props = {
+    urn: string;
+    id: string;
+};
+
+type EntityTypeCount = {
+    count: number;
+    displayName: string;
+};
+
+/**
+ * Extract entity type counts to display in the ingestion summary.
+ *
+ * @param entityTypeFacets the filter facets for entity type.
+ * @param subTypeFacets the filter facets for sub types.
+ */
+const extractEntityTypeCountsFromFacets = (
+    entityTypeFacets: FacetMetadata,
+    subTypeFacets?: FacetMetadata | null,
+): EntityTypeCount[] => {
+    const finalCounts: EntityTypeCount[] = [];
+
+    if (subTypeFacets) {
+        subTypeFacets.aggregations.forEach((agg) =>
+            finalCounts.push({
+                count: agg.count,
+                displayName: pluralize(agg.count, capitalizeFirstLetterOnly(agg.value) || ''),
+            }),
+        );
+        entityTypeFacets.aggregations
+            .filter((agg) => !ENTITIES_WITH_SUBTYPES.has(agg.value.toLowerCase()))
+            .forEach((agg) =>
+                finalCounts.push({
+                    count: agg.count,
+                    displayName: pluralize(agg.count, capitalizeFirstLetterOnly(agg.value) || ''),
+                }),
+            );
+    } else {
+        // Only use Entity Types- no subtypes.
+        entityTypeFacets.aggregations.forEach((agg) =>
+            finalCounts.push({
+                count: agg.count,
+                displayName: pluralize(agg.count, capitalizeFirstLetterOnly(agg.value) || ''),
+            }),
+        );
+    }
+
+    return finalCounts;
+};
+
+export default function IngestedAssets({ urn, id }: Props) {
+    // First thing to do is to search for all assets with the id as the run id!
+    console.log(urn);
+
+    const [showAssetSearch, setShowAssetSearch] = useState(false);
+
+    // Execute search
+    const { data, loading, error } = useGetSearchResultsForMultipleQuery({
+        variables: {
+            input: {
+                query: '*',
+                start: 0,
+                count: 1,
+                filters: [
+                    {
+                        field: 'runId',
+                        value: id,
+                    },
+                ],
+            },
+        },
+    });
+
+    // Parse filter values to get results.
+    const facets = data?.searchAcrossEntities?.facets;
+    const entityTypeFacets = facets?.filter((facet) => facet.field === 'entity')[0];
+
+    const hasSubTypeFacet = (facets?.findIndex((facet) => facet.field === 'typeNames') || -1) >= 0;
```

Comment:
real nit: I believe using `indexOf` will return `-1` if it can't find the item, just as a shortcut for you

Reply:
indexOf only supports comparing the immediate object right? Here we want to check a subfield of the facet

Reply:
ohh true my bad - ignore!